### ISSUE TITLE
Sats through HC Jiraiya desapear

### ISSUE THREAD

**Issue by AbadeC** [nbd-wtf/obw#50]
When sending Sats to OBW using HC Jiraiya, the amount is debited from the sender but never reaches the destination. It remains "waiting" forever and get lost. 

I tested it on March 22nd and today, April 11th, and the behavior is the same. If the problem is in HC Jiraiya, 

I suggest removing it from the list of Hosted Channel Providers, as remaining causes users to lose precious sats.

**Comment by fiatjaf** [comment]
Can you message me on https://t.me/fiatjaf?

**Comment by AbadeC** [comment]
From past week on, SATM doesn't work neither. Sats don't enter: "waiting" forever, as Jiraiya. Beside that, if I try send Sats, they go awya, but don't reach destination. Strange, but this problem with HC seems to happen only with me. Restart OBW: no effect. If I uninstall OBW, will I be able to recover the HC if I inform the seed?

**Comment by Darth-Coin** [comment]
Yes, if you remove OBW and install it again all the HC balance remain, just go and add the HC again after restore and your channel and sats will be there.
Be sure your OBW is syncing correctly. So when you open it, wait few moments to see if appear on top the sign that channels are syncing. If the Electrum connection is not ready, channels will not sync, so you cannot make any payments.

**Comment by AbadeC** [comment]
> Yes, if you remove OBW and install it again all the HC balance remain, just go and add the HC again after restore and your channel and sats will be there. Be sure your OBW is syncing correctly. So when you open it, wait few moments to see if appear on top the sign that channels are syncing. If the Electrum connection is not ready, channels will not sync, so you cannot make any payments. You could ask further questions also in OBW Telegram group https://t.me/openbitcoinwallet

I've installed OBW in *another* device, with the seed and backup file from the first one. The Hosted Channel SATM was showed up as expected, with correct balance. But the problem persists: each receiving implies in "waiting" state... forever. Meanwhile the sender is always debited, as expected. The only problem is that it doesn't reach its destination. The same with Jiraiya.
